service-front/module/Application/view/application/pages/passport_number_fail.twig

```diff
@@ -25,11 +25,11 @@
             {{ passport_indate }}
         </p>
         <a href="./passport-number" class="govuk-button">Edit details</a>
-        <p class="govuk-body">
+{#        <p class="govuk-body">#}
```

Comment:
Was this intentional? I'd think the links should be wrapped in some sort of block element, whether that's a <div> or a <p>